### ISSUE TITLE
A way to make a script

### ISSUE THREAD

**Issue by Loup-Garou911XD** [KhunHtetzNaing/ADB-OTG#16]
Can you please make it so we can make a script,or can you suggest how to add a script in the code

**Comment by Loup-Garou911XD** [comment]
Or can you please add a script and compile the app for me please

**Comment by PleaseHelpBe** [comment]
Yes that script creation thing is biggie considering it's what I downloaded this app to do

**Comment by Loup-Garou911XD** [comment]
> Yes that script creation thing is biggie considering it's what I downloaded this app to do

You can do scripting in bash though,just have to copy paste it in the app

**Comment by PleaseHelpBe** [comment]
I am not actually involved in any of this conversation and neither have I developed anything at as I have no knowledge of how. It's clear that Scotty is still using my name to attempt to frame me or make it seem like I am involved

**Comment by PleaseHelpBe** [comment]
I'm Bryan Eastham and the hacker is my fraternal twin bro Scott Eastham. But if you have been communicating with him, it's very likely you already knew this

**Comment by Loup-Garou911XD** [comment]
What are you talking about?
